bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evcc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="bridge">
+		<label>evcc Bridge</label>
```

Comment:
Similar for the label. No need to use the 'evcc' prefix in any of the bridges/things